.github/workflows/release.yml

```diff
@@ -216,6 +155,49 @@ jobs:
           if [ "${{ inputs.dry_run }}" = "true" ]; then  
             box_summary="[Dry run] "
           fi
+              echo "$box_summary :ballot_box: ${{ inputs.release_version }} was uploaded to https://pentaho.app.box.com/folder/$BOX_FOLDER_ID" >> $GITHUB_STEP_SUMMARY
```

Comment:
Can't we align the `echo` with the `if`?

Reply:
aligned